components/payment-term.jsx

```diff
@@ -107,11 +108,24 @@ export function PaymentTerm({
 		},
 		monthly: {
 			title: 'Monthly',
-			price: (price) => (
-				<React.Fragment>
-					<span className="ncf__payment-term__price">{price}</span> per month
-				</React.Fragment>
-			),
+			price: (price, is7DayPassExperiment) => {
+				const paymentIntervalTextToDisplay = (() => {
+					switch (true) {
+						case is7DayPassExperiment:
+							return ' one-time payment';
+
+						default:
+							return ' per month';
+					}
```

Comment:
nitpicking? non-blocking:
I'm not sure about the benefits of the switch construct here.
This comment https://github.com/Financial-Times/n-conversion-forms/pull/751/files#r1308857637 makes me think this is better.
The following should be the same, but arguably easier to read (less language construct).
```suggestion
				if (is7DayPassExperiment) {
						return ' one-time payment';
				}
				return ' per month';
```

As mentioned in the next-subscribe PR, that mostly personal preference and is strictly identical, I just want to ensure that we use what people are more familiar with, in the spirit of "Write code you can fix at 3AM".
But then again, @AniaMakes learned something new, so why not! :)

Reply:
I know what you mean, Ludo, about "fix at 3AM" but for me it was a very quick "huh", a google to confirm it does what I think it does, and "oh, that's neat!". I think it would be beneficial to many long term and could avoid some gnarly if-else or multi-step ternaries.